test/gui/shared/scripts/pageObjects/Activity.py

```diff
@@ -135,3 +173,64 @@ def hasSyncStatus(filename, status):
             return False
         except:
             return False
+
+    @staticmethod
+    def select_account(account):
+        squish.clickButton(squish.waitForObject(Activity.LOCAL_ACTIVITY_FILTER_BUTTON))
+        squish.activateItem(
+            squish.waitForObjectItem(Activity.ACCOUNT_SELECTOR, account)
+        )
+
+    @staticmethod
+    def get_synced_file_selector(resource):
+        return {
+            "column": 1,
+            "container": Activity.SYNCED_TABLE,
+            "text": resource,
+            "type": "QModelIndex",
+        }
+
+    @staticmethod
+    def check_table_content(action, resource, account):
+        try:
+            fileRow = squish.waitForObject(
+                Activity.get_synced_file_selector(resource),
+                get_config('lowestSyncTimeout') * 1000,
+            )["row"]
+            squish.waitForObjectExists(
+                {
+                    "column": 0,
+                    "row": fileRow,
+                    "container": Activity.SYNCED_TABLE,
+                    "text": action,
+                    "type": "QModelIndex",
+                }
+            )
+            squish.waitForObjectExists(
+                {
+                    "column": 4,
```

Comment:
is it possible to to get the column number by the name? so that we can be sure that we are check on correct column